hydromt_wflow/workflows/landuse.py

```diff
@@ -11,7 +11,7 @@
 __all__ = ["landuse", "lai"]
 
 
-RESAMPLING = {"landuse": "nearest", "lai": "average"}
+RESAMPLING = {"landuse": "nearest", "lai": "average", "alpha_h1": "mode"}
 DTYPES = {"landuse": np.int16}
```

Comment:
Maybe also here add the dtype in case users "forgot" in their catalog (or did not do a catalog)?
```suggestion
DTYPES = {"landuse": np.int16, "alpha_h1": np.int16}
```

Reply:
Ah thanks for this suggestion, this is indeed a nice option to include!